README.md

```diff
@@ -306,6 +306,21 @@ import { setClassNamePrefix } from 'ui-box'
 setClassNamePrefix('📦')
 ```
 
+### Safe `href`s
+
+By default `ui-box` does not ensure that safe urls are used when passed to an element. But we built this functionality into `ui-box` to  protect the end users of the products you are building. You can alter this by using `setUseSafeHref(boolean)`. This will ensure that only safe protocols are used (`http:`, `https:`, `mailto:`, `tel:`, and `data:`) and that the correct `rel` values are added (`noopener`, `noreferrer`(for external links)).
```

Comment:
I think the functionality is clear from the docs, but we may want to describe it as a [scheme](https://en.wikipedia.org/wiki/URL#Syntax), instead of a url (`ensure that safe urls`).

Reply:
makes sense